plugins/fabric8-multi-tenant-manager/src/main/java/com/redhat/che/multitenant/Fabric8OpenShiftProjectFactory.java

```diff
@@ -69,6 +72,26 @@ public String evaluateLegacyNamespaceName(NamespaceResolutionContext resolutionC
     return evaluateNamespace();
   }
 
+  @Override
+  protected String evalPlaceholders(String namespace, NamespaceResolutionContext ctx) {
+    return evalPlaceholders();
+  }
+
+  @Override
+  protected String evalPlaceholders(String namespace, Subject currentUser, String workspaceId) {
+    return evalPlaceholders();
+  }
+
+  private String evalPlaceholders() {
+    String placeholder = null;
+    try {
+      placeholder = evaluateNamespace();
+    } catch (InfrastructureException e) {
+      LOG.error("Failed to evaluate placeholder for '*-che' namespace: {}", e.getMessage());
+    }
+    return placeholder;
```

Comment:
I'm not sure that returning null here won't cause NPE above in upstream code.

Reply:
technically this should never happen, I can return emty string though if you think this would be better